ui/src/pages/Tenant/Detail/NewBackup/index.tsx

```diff
@@ -250,6 +253,61 @@ export default function NewBackup() {
                     />
                   </Form.Item>
                 </Col>
+
```

Comment:
Input form field `Host` should be added to the form if `destType` is not `NFS`. Refer to PRD to see the details.

Reply:
It was originally there
<img width="1236" alt="image" src="https://github.com/user-attachments/assets/524d0e2d-b988-41cc-ab5a-dda07c7d9812">

Reply:
I mean that a new field named `Host` should be added to the form when selecting COS, OSS, S3 or S3-Compatible.

Reply:
done